### ISSUE TITLE
Configuration to use max-line-length for strings

### ISSUE THREAD

**Issue by joakimthorell** [pinterest/ktlint#2975]
## Expected Behavior

From #2480 their was a request to add configuration to skip `max-line-length` for long strings. But this never became a configuration but instead always skipping strings. 
It would be nice to have some override to allow this rule for long strings as well.

```kotlin
log.info {
    "Long line of logging you might want to have on multiple lines " +
    "so that it not succeeds the max-line-length"
}
```

## Current Behavior

It would allow a string such as
```kotlin
log.info {
    "Long line of logging you might want to have on multiple lines so that it not succeeds the max-line-length"
}
```

## Additional information
* Current version of ktlint: 1.5.0

**Comment by paul-dingemans** [comment]
No configuration option will be added for this. Please see https://pinterest.github.io/ktlint/latest/faq/#can-a-new-toggle-be-added-to-optionally-enabledisable-format-code-in-a-particular-way.
